packages/roosterjs-editor-dom/lib/edit/adjustInsertPosition.ts

```diff
@@ -235,9 +242,36 @@ function adjustInsertPositionForMoveCursorOutOfALink(
     return position;
 }
 
+/**
+ * Adjust the cursor position when a image is inserted to prevent it be inserted into another element.
+ */
+function adjustInsertPositionForImages(
+    root: HTMLElement,
+    nodeToInsert: Node,
+    position: NodePosition,
+    range: Range
+): NodePosition {
+    const isImage = nodeToInsert.nodeName === 'IMG' ? true : false;
```

Comment:
When you want to get a boolean value, just directly retrieve from the condition, no need to have "true : false" at the end:

```ts
const isImage = nodeToInsert.nodeName === 'IMG';
```

And You can call getTagOfNode() to make it safer. It can handle all kind of input node including null.